src/opendr/perception/pose_estimation/hr_pose_estimation/algorithm/scripts/make_val_subset.py

```diff
@@ -0,0 +1,43 @@
+import json
```

Comment:
from the last update the directory algorithm/scripts is imported from the light weight open pose tool so it is used on that tool, but if we could upload a high-resolution dataset on the OpenDR server it could also be useful for this tool also